build.gradle.kts

```diff
@@ -30,7 +30,7 @@ dependencies {
 
 // Set the JVM language level used to build the project. Use Java 11 for 2020.3+, and Java 17 for 2022.2+.
```

Comment:
The comment here use Java11 for 2020.3+, why change to Java17 here

Reply:
The comment here use Java 17 for 2022.2+.